pubsub/callback/callback.go

```diff
@@ -0,0 +1,41 @@
+package callback
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/micromdm/micromdm/pubsub"
+)
+
+type CallbackPublisher struct {
+	url  string
+	next pubsub.Publisher
+}
+
+func (p *CallbackPublisher) Publish(ctx context.Context, topic string, msg []byte) error {
+
+	// send to callback url
+	var buf bytes.Buffer
+	var event = struct {
+		Topic   string `json:"topic"`
+		Message []byte `json:"message"`
```

Comment:
Well, it'll turn it into a b64 byte array. I'd want to avoid that. I'd want json elements for all the command response dictionaries and keys and whatnot (just like the command-submit API) for this _particular_ API. Otherwise you have to mess with decoding and other nonsense when all you really want is the command responses in JSON.